requests-for-comments/2025-01-29-startups-workflow.md

```diff
@@ -0,0 +1,37 @@
+# Request for comments: Startups workflow streamlining
+
+## Problem statement
+We need to streamline the startup application and approval workflow. It’s a multidisciplinary job and we need some hands who good at integrations (in between Zapier, Stripe, Vitally, Slack and PostHog-the-app and PostHog.com). Doing this will serve a core audience better, increase our speed.
+
+Right now, when startups and YC members apply for our programs and deals, it's a manual process that results in friction, delay, data loss, missed opportunities, bad reporting and general inefficiency.
+
+## What's wrong with current flow:
+- **It's slow for founders:** Manual effort, email-based (which we hate), asking for duplicate info, all of which gets in the way of their JTBD (shipping). 
+- **It's slow for us:** This program matters to PostHog, and this is unnecessary friction. (Like, poor Scott. There's at least 30 of these a day.)
+
+## How the current flow works:
+1. People fill out the forms on https://posthog.com/startups 
+2. An email (via Zap) goes out from Scott Lewis, asking participants to manually submit their info
+3. Participants manually submit, over email, the date they were founded, their project ID, last 4 digits of their CC, amount they've fundraised, the email address associated with their PostHog account
+4. Over email, Scott receives this, manually reviews the info across Stripe and Vitally to see if they are eligible, and to check whether they have the credits or not.
+5. He then presses Approve in Zapier (which applies the credits in Stripe, and triggers the next workflow of emailing the person with their update + merch stuff)
+
+## What can we do about it? 
+- **Steps 2 and 3,** above, should not have to happen. (We already have a record of *most* of the info, because a PostHog account is required to apply.)
+- I think this means the actual **sign-up form should be gated**, accessible from the app itself — requiring a user to be in a logged-in state to submit. 
+- I think the system should **auto-filter applications:** if they are not eligible, do not apply the credits. *(We can write nice rejections: “Did we get that wrong? Contact Scott if you think there’s been a mistake. We like being generous, but sometimes the our auto-bots are fierce defenders of the realm.”)*
+- I think the system should **auto-approve people:** if they pass the above criteria check, the credits should simply be applied. We can be alerted to abuses if needed, starting with daily manual reviews. 
+
+## Proposed flow:
+ * Remove the actual application form fields (just the form fields) from [the page](https://posthog.com/startups/apply)
+* Replace it with a call-to-action to “Sign in or sign up to apply”
+* Use PostHog-the-app to filter by whether a user has clicked that or not, and show a banner that says “🎉 You’re eligible for the startup plan! Get started here.”
```

Comment:
We don't really have this data available in app so maybe it's available to everyone somewhere on the billing page?